### ISSUE TITLE
iTMSTransporter moved in Xcode 6.3

### ISSUE THREAD

**Issue by grzesiekko** [nomad/shenzhen#243]
In the most recent version of Xcode iTMSTransporter has been moved from:

```
Xcode.app/Contents/Applications/Application\ Loader.app/Contents/MacOS/itms/bin/iTMSTransporter
```

to:

```
Xcode.app/Contents/Applications/Application\ Loader.app/Contents/itms/bin/iTMSTransporter
```

**Comment by silvansky** [comment]
Hey, why don't you accept a fix in PR? I think this is an important thing.

**Comment by mattt** [comment]
Fixed by #245.
